tests/testthat/test-edit.R

```diff
@@ -68,6 +72,16 @@ test_that("edit_r_profile() ensures .Rprofile exists in project", {
   expect_proj_file(".Rprofile")
 })
 
+test_that("edit_r_buildignore() ensures .Rbuildignore exists in project", {
```

Comment:
Here you're effectively just testing `edit_file()`, so I think you can remove the test.